app/controllers/api/watches/toggle_controller.rb

```diff
@@ -9,11 +9,15 @@ def index
   end
 
   def create
-    @watch = Watch.create!(
-      user: current_user,
-      watchable: watchable
-    )
-    render json: @watch
+    if Watch.exists?(user_id: current_user.id, watchable_id: params[:watchable_id], watchable_type: params[:watchable_type])
+      render json: { message: "この#{watchable.class.model_name.human}はWatch済です。" }, status: :unprocessable_entity
+    else
+      @watch = Watch.create!(
+        user: current_user,
+        watchable: watchable
+      )
+      render json: @watch
```

Comment:
```suggestion
      watch = Watch.create!(
        user: current_user,
        watchable: watchable
      )
      render json: watch
```

jsonでrenderするならインスタンス変数にする必要はないかも？